src/_guides/language/language-tour.md

```diff
@@ -3197,11 +3242,11 @@ class SmartTelevision [!extends!] Television {
 {% endprettify %}
 
 
-
+<a name="overridable-operators"></a>
```

Comment:
Could this be spurious? I can't see other occurrences of `<a name="...">` in this file (having expanded the whole file here, and also searching in https://raw.githubusercontent.com/dart-lang/site-www/master/src/_guides/language/language-tour.md), and it's surprising that, if I understand this mechanism correctly, a click on 'operators' will take us to this point, showing a section called 'Overriding members'.

Reply:
This ensures that links, from outside the site, that target `#overridable-operators` will still work. 

I can certainly drop this if you prefer. @kwalrath?

Reply:
OK, then it makes sense to keep it. But maybe the link from the same file shouldn't have the link text 'operators' when it jumps to 'Overriding members'.

Reply:
Oh, thanks for pointing that out! I missed the in-page link to `#overridable-operators` (strange, I thought that I had searched for those before); that needs to be changed to `#_operators`.

Hmm, I've also found another place in the tour that needed a slight update. Please see my latest commit.